workflow/applications/gfs_cycled.py

```diff
@@ -89,9 +89,6 @@ def _get_app_configs(self):
         if self.do_awips:
             configs += ['awips']
 
-        if self.do_npoess:
-            configs += ['npoess']
```

Comment:
Why is this being removed?

Reply:
See https://github.com/NOAA-EMC/global-workflow/pull/2103#discussion_r1415841460

Reply:
Fine doing away with the switch, but you still have to add the config file where the other gempak config is being loaded.